hd/lang/lex_utf8.txt

```diff
@@ -10,20 +10,6 @@ fr: Tapez à l'identique le nom de la généalogie que vous voulez utiliser :
 en: list of available genealogies (use gwsetup)
 fr: listes des généalogies disponibles (utilise gwsetup)
 
-    about (short date)
```

Comment:
For next time, I don't understand this modification. It seems it is not related to this PR.